demo/controllers/availableComponentsController.ts

```diff
@@ -0,0 +1,28 @@
+///
+/// Copyright 2015-2016 Red Hat, Inc. and/or its affiliates
+/// and other contributors as indicated by the @author tags.
+///
+/// Licensed under the Apache License, Version 2.0 (the "License");
+/// you may not use this file except in compliance with the License.
+/// You may obtain a copy of the License at
+///
+///    http://www.apache.org/licenses/LICENSE-2.0
+///
+/// Unless required by applicable law or agreed to in writing, software
+/// distributed under the License is distributed on an "AS IS" BASIS,
+/// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+/// See the License for the specific language governing permissions and
+/// limitations under the License.
+///
```

Comment:
I would rather we didn't have headers for licensing on every single file.  It makes maintaining them a pain (particularly for the copyright dates), and the LICENSE at the root of the repo is sufficient as per our lawyer.  See what we did in the ManageIQ/manageiq repo.

Additionally, the license should specify "ManageIQ Authors", not "Red Hat" as per the license in ManageIQ.